src/util/ApplyValueTransformers.ts

```diff
@@ -0,0 +1,21 @@
+import { ValueTransformer } from "../decorator/options/ValueTransformer";
+
+export class ApplyValueTransformers {
+    static innerTransform(transformer: ValueTransformer | ValueTransformer[], databaseValue: any) {
```

Comment:
I am not sure about this methods names `inner`, `outer`. Maybe `toTransform`, `fromTransform` ? It's just my opinion. I am good with this if other says it is ok :)

Reply:
I wasn't very inspired, `inner` and `outer` are not clear ^^. I'll go with your opinion

Reply:
`transformFrom` and `transformTo` are even clearer